### PR TITLE
Visual fixes to ErrorBody Page

### PR DESCRIPTION
**Pull request by jay-khatri**
### Remove bottom border on ErrorBody `Stat`
There's a weird bottom border on the error body `stat` section. It looks like this:
image

This change removes the bottom border on each `stat` component. It now looks like:
image

### Adds a border radius to the "Instance Error Body" section, see below:
image
previously, there was no radius.

## How did you test this change?

Visual test.

## Are there any deployment considerations?
no

### CODE REVIEW

File: .reflame.config.jsonc [hunk 1]
```diff
@@ -45,19 +45,19 @@
 			"HEIGHT_CLIENT_ID": "w8GzB4GI8gGUE2iEAQMY8Wck44gK2ek2i4Q8q4myKQY",
 			"DEMO_SESSION_URL": "/1/sessions/SGKFqG4mE8au2UmHjT0BhjO5Q15f",
 			"LINEAR_CLIENT_ID": "f60ff43c7376d0aceaa1e111db39e60d",
-			"DEMO_PROJECT_ID": "1344",
+			"DEMO_PROJECT_ID": "1344"
```
Comment (jay-khatri) [inline .reflame.config.jsonc:48]: @Vadman97 @ccschmitz what should I do here?

Comment (ccschmitz) [reply .reflame.config.jsonc:48]: I get these too and wasn't sure why. I've just been committing them as I don't think they really impact anything. Maybe a conflict with our editor settings and someone else's.

**Review by ccschmitz — APPROVED** [review]
😻

File: .reflame.config.jsonc (hunk 1 above)
Comment (et) [reply .reflame.config.jsonc:48]: You need the trailing comma. Run `yarn format:all` to fix. 

We might need to tune something in reflame that's injecting these (or maybe it's from vscode). @lewisl9029 - any ideas?

Comment (jay-khatri) [reply .reflame.config.jsonc]: thanks.

Comment (lewisl9029) [reply .reflame.config.jsonc]: Ah yep, that's my bad, I forgot to update the env script after https://github.com/highlight/highlight/pull/5571

Thanks for the ping @et, will put up a PR to fix asap!

Comment (lewisl9029) [reply .reflame.config.jsonc]: Here's the PR: https://github.com/highlight/highlight/pull/5641

**Pull request merged**